contrib/cmd/fs-idmap/fs-idmap.go

```diff
@@ -31,20 +31,22 @@ func main() {
 	if err := cmd.Start(); err != nil {
 		log.Fatalf("failed to run the helper binary: %v", err)
 	}
+	defer func() { _ = cmd.Wait() }()
 
 	path := fmt.Sprintf("/proc/%d/ns/user", cmd.Process.Pid)
 	var userNsFile *os.File
 	if userNsFile, err = os.Open(path); err != nil {
 		log.Fatalf("unable to get user ns file descriptor: %v", err)
-		return
 	}
 	defer userNsFile.Close()
 
 	attr := unix.MountAttr{
 		Attr_set:  unix.MOUNT_ATTR_IDMAP,
 		Userns_fd: uint64(userNsFile.Fd()),
 	}
-	if err := unix.MountSetattr(treeFD, "", unix.AT_EMPTY_PATH|unix.AT_RECURSIVE, &attr); err != nil {
+	if err := unix.MountSetattr(treeFD, "", unix.AT_EMPTY_PATH, &attr); err != nil {
 		log.Fatalf("error calling mount_setattr: %v", err)
```

Comment:
Same comment regarding `Fatalf`.

Reply:
Solved in the same way (moved this code to another function, so call defer there and os.Exit(1) just in main), thanks!